articles/Remote Debugging/how-to-remote-debugging-with-vorlonJS.md

```diff
@@ -0,0 +1,33 @@
+<!--
+   tags:
+       Debugging:
+           - "Remote debugging with VorlonJS"
+-->
+# Remote debugging with VorlonJS
+
+> This article explains how to remotely debugging and testing your JavaScript using `vorlon.js`. While working with a Mac and an iPad might be straight forward, achievieving the same thing with a PC is more complicate. Here is one possible approach.
+
```

Comment:
I would expand the intro a bit more. Say what kind of debugging we can do with Vorlon - DOM inspection, console output, CSS inspection. Can it do breakpoints or step-throughs? Can we persist or save our code changes? Say that it is specifically useful for developing on mobile devices.

Reply:
Don't forget to include a link to their website or any relevant documentation.